container/libcontainer/compatibility.go

```diff
@@ -144,6 +144,40 @@ type preAPINetwork struct {
 	TxQueueLen int `json:"txqueuelen,omitempty"`
 }
 
+type oldCgroup struct {
```

Comment:
nit: old doesn't really mean anything once we face this issue again. 
Shall we rename this to v1Cgroup or something similar to that? Let's also add a few comments describing the versions of docker this version is compatible with.